volatility3/framework/plugins/windows/writecrashdump.py

```diff
@@ -0,0 +1,134 @@
+from typing import List, Type, Iterator, Tuple
+
+from volatility3.framework import interfaces, renderers, constants, exceptions
+from volatility3.framework.configuration import requirements
+from volatility3.framework.interfaces import configuration, plugins
+from volatility3.framework.layers import physical, intel
+from volatility3.framework.symbols import intermed
+from volatility3.plugins.windows import info
+
+class WriteCrashDump(plugins.PluginInterface):
+    """Runs the automagics and writes the output to a crashdump format file"""
+
+    _required_framework_version = (2, 0, 0)
+    _version = (2, 0, 0)
+
+    @classmethod
+    def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]:
+        return [
+            requirements.TranslationLayerRequirement(name = 'primary', description = 'Memory layer for the kernel',
+                                                     architectures = ["Intel32", "Intel64"]),
+            requirements.SymbolTableRequirement(name = "nt_symbols", description = "Windows kernel symbols"),
+            requirements.VersionRequirement("info", component = info.Info, version = (1, 0, 0))
+        ]
+
+    @classmethod
+    def get_physical_layer_name(cls, context: interfaces.context.ContextInterface, vlayer: interfaces.layers.DataLayerInterface) -> str:
+        return context.config.get(interfaces.configuration.path_join(vlayer.config_path, 'memory_layer'), None)
+
+    @classmethod
+    def write_crashdump(cls, context: interfaces.context.ContextInterface, layer_name: str, symbol_table: str,
+                        open_method: Type[interfaces.plugins.FileHandlerInterface]):
+        kvo = context.layers[layer_name].config["kernel_virtual_offset"]
+        kernel = context.module(symbol_table, layer_name = layer_name, offset = kvo)
```

Comment:
This would become simpler and not need error checking if the kernel were a `ModuleRequirement` above, otherwise this should have checks for the kvo not existing

Reply:
I have replaced the requirements with a module requirement